src/state_machine/coordinator/fire.rs

```diff
@@ -490,8 +490,9 @@ impl<Aggregator: AggregatorTrait> Coordinator<Aggregator> {
             self.dkg_public_shares
                 .insert(dkg_public_shares.signer_id, dkg_public_shares.clone());
             debug!(
-                "DKG round {} DkgPublicShares from signer {}",
-                dkg_public_shares.dkg_id, dkg_public_shares.signer_id
+                dkg_id = %dkg_public_shares.dkg_id,
+                signer_id = %dkg_public_shares.signer_id,
+                "DKG round DkgPublicShares received"
```

Comment:
I think the text here can be just `DkgPublicShares received`, because we will add `dkg_id = ` immediately after.